components/Tabs/Tabs.js

```diff
@@ -1,62 +1,80 @@
+// I feel like I made this overly complicated, but oh well ¯\_(^_^)_/¯
 
-class TabLink {
-  constructor(element) {
-    // Assign this.element to the passed in DOM element
-    // this.element;
-    
-    // Get the custom data attribute on the Link
-    // this.data;
-    
-    // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
-    // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
-    
-    // Add a click event listener on this instance, calling the select method on click
-
-  };
-
-  select() {
-    // Get all of the elements with the tabs-link class
-    // const links;
-
-    // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
-
-    // Add a class named "tabs-link-selected" to this link
-    // this.element;
-    
-    // Call the select method on the item associated with this link
+class Tabs {
+  constructor (element) {
+    // assign this.element to the tabs container
+    this.element = element;
+    // collect the tab links
+    this.links = this.element.firstElementChild.children;
+    // collect the tab items
+    this.items = this.element.lastElementChild.children;
```

Comment:
nicely done with the inherent node properties here instead of either going back to the DOM or doing a querySelectorAll on the element being passed in